### PR TITLE
Added majorUnits, minorUnits, majorPart and minorPa…

### PR DESCRIPTION
**Pull request by sanmibuh**
…rt utils

### CODE REVIEW

File: src/groovy/com/ticketbis/money/Money.groovy [hunk 1]
```diff
@@ -57,6 +57,15 @@ final class Money implements Serializable, Comparable<Money>, MoneyExchange, Mon
         this.currency = currency
     }
 
+    public static Money fromMinorUnits(Number minorAmount, Currency currency) {
+        BigDecimal amount = ((BigDecimal)minorAmount).scaleByPowerOfTen(-currency.defaultFractionDigits)
```
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: Code style

File: src/groovy/com/ticketbis/money/Money.groovy [hunk 2]
```diff
@@ -208,6 +217,64 @@ final class Money implements Serializable, Comparable<Money>, MoneyExchange, Mon
...
+    /**
+    * Gets the amount major part as a {@code long}.
+    * <p>
+    * This returns the Money amount in terms of the major units of the currency, truncating the
+    * amount if necessary. For example, 'EUR 2.35' will return 'EUR 2', and 'BHD -1.345' will
+    * return 'BHD -1'.
+    * <p>
+    * @return the major units part of the amount
+    */
+    public long majorPart(){
```
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: Code style

File: src/groovy/com/ticketbis/money/Money.groovy [hunk 3]
```diff
@@ -208,6 +217,64 @@ final class Money implements Serializable, Comparable<Money>, MoneyExchange, Mon
...
+    /**
+     * Extract the minor part as a {@code long}.
+     * <p>
+     * This returns the Money amount in terms of the minor units of the
+     * currency, truncating the whole part if necessary. For example, 'EUR 2.35'
+     * will return 35, and 'BHD -1.345' will return -345.
+     * <p>
+     * @return the minor units part of the amount
+     */
+    public long minorPart(){
```
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: Code style

File: src/groovy/com/ticketbis/money/Money.groovy [hunk 4]
```diff
@@ -208,6 +217,64 @@ final class Money implements Serializable, Comparable<Money>, MoneyExchange, Mon
...
+     * Gets the amount in major units as a {@code long}.
+     * <p>
+     * This returns the Money amount in terms of the major units of the
+     * currency, truncating the amount if necessary. For example, 'EUR 2.35'
+     * will return 2, and 'BHD -1.345' will return -1.
+     * <p>
+     *
+     * @return the major units of the amount
+     */
+    public long majorUnits() {
```
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: getMayorUnits?

File: src/groovy/com/ticketbis/money/Money.groovy [hunk 5]
```diff
@@ -208,6 +217,64 @@ final class Money implements Serializable, Comparable<Money>, MoneyExchange, Mon
...
+     * Gets the minor part as a {@code long}.
+     * <p>
+     * This returns the Money amount in terms of the minor units of the
+     * currency, truncating the whole part if necessary. For example, 'EUR 2.35'
+     * will return 'EUR 235', and 'BHD -1.345' will return 'BHD -1345'.
+     * <p>
+     *
+     * @return the minor units of the amount
+     */
+    public long minorUnits() {
```
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: getMinorUnits?

File: src/groovy/com/ticketbis/money/Money.groovy (hunk 3 above)
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: getMinorPart?

File: src/groovy/com/ticketbis/money/Money.groovy (hunk 2 above)
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: getMayorPart?

File: src/groovy/com/ticketbis/money/Money.groovy [hunk 6]
```diff
@@ -57,6 +57,15 @@ final class Money implements Serializable, Comparable<Money>, MoneyExchange, Mon
         this.currency = currency
     }
 
+    public static Money fromMinorUnits(Number minorAmount, Currency currency) {
```
Comment (jonan) [inline src/groovy/com/ticketbis/money/Money.groovy]: I think the new constructors should be documented

File: src/groovy/com/ticketbis/money/Money.groovy (hunk 4 above)
Comment (sanmibuh) [inline src/groovy/com/ticketbis/money/Money.groovy]: It is new JavaMoney/jsr354-r API method name

File: src/groovy/com/ticketbis/money/Money.groovy (hunk 5 above)
Comment (sanmibuh) [inline src/groovy/com/ticketbis/money/Money.groovy]: It is new JavaMoney/jsr354-r API method name

File: src/groovy/com/ticketbis/money/Money.groovy (hunk 2 above)
Comment (sanmibuh) [inline src/groovy/com/ticketbis/money/Money.groovy]: It is new JavaMoney/jsr354-r API method name

File: src/groovy/com/ticketbis/money/Money.groovy (hunk 3 above)
Comment (sanmibuh) [inline src/groovy/com/ticketbis/money/Money.groovy]: It is new JavaMoney/jsr354-r API method name

**Comment by sanmibuh** [comment]
Code style revised and removed version update

**Pull request merged**